media/api.go

```diff
@@ -5,11 +5,14 @@ import (
 	"fmt"
 	"io/ioutil"
 	"net/http"
+
+	"github.com/sirupsen/logrus"
```

Comment:
This is a minor thing, but we've been importing logrus as `log` in go code as a convention.

```log "github.com/sirupsen/logrus"```

This may have gotten through in other places too.